stream/stream.go

```diff
@@ -184,3 +149,61 @@ func registerGob() {
 	})
 	rpc.RegisterGob()
 }
+
+func NewSamplingSender(next zoekt.Sender) *samplingSender {
```

Comment:
minor: the docstring on the exported function is more useful here than on samplingSender struct.

Reply:
thanks!